core/src/main/java/org/apache/calcite/plan/hep/HepRelVertex.java

```diff
@@ -75,8 +76,8 @@
     return currentRel.getRowType();
   }
 
-  @Override protected String computeDigest() {
-    return "HepRelVertex(" + currentRel + ")";
+  @Override protected Digest computeDigest() {
+    return Digest.create(this, getRelTypeName() + '#' + getCurrentRel().getId());
   }
 
```

Comment:
Why do you use `getRelTypeName()` rathen than "HepRelVertex"?

Reply:
To unify the logic, we can switch to "HepRelVertex", that's a preference.